pyproject.toml

```diff
@@ -105,3 +106,19 @@ filterwarnings = "ignore:Development network has a block height of"
 [tool.coverage.run]
 source = ["beamer"]
 omit = ["beamer/tests/*"]
+
+[tool.ruff]
+exclude = [
+    "build",
+    "dist",
+    ".git",
+    ".mypy_cache",
+    "__pycache__",
+]
```

Comment:
Nitpick: I think we should sync this list with the one in `[tool.black]` section, and also remove `build` from both because it shouldn't be relevant (anymore).